README.md

```diff
@@ -198,7 +205,7 @@ Docker
 
 ```bash
 # build an image with catapult installed
-docker build --target=catapult -t release-resource -f Dockerfile .
+docker build --target=catapult -t catapult -f Dockerfile .
```

Comment:
Not sure if this is right?

Reply:
This is specifically for how to build the Concourse resource. The naming doesn't strictly matter since you can reference any arbitrary docker repo in the concourse pipeline definition, but all our pipelines reference one called `release-resource` - maybe it would be better as `catapult-release-resource` or something (these seem to conventionally have the `-resource` suffix).

Reply:
There's two `docker build` commands in the README, and both tagged `release-resource`

>  ```bash
>  # build an image with catapult installed
>  docker build --target=catapult -t release-resource -f Dockerfile .
>  
>  # build an image for the concourse resource
>  docker build --target=release-resource -t release-resource -f Dockerfile .
>  ```

It felt like the first one was to build the CLI tool into a docker image. I didn't want to go down the Concourse rabbit hole today, so I can just change this back?

Reply:
Oh I missed that. Then let's keep your change.